keyboards/converter/periboard_512/rules.mk

```diff
@@ -0,0 +1,29 @@
+# MCU name
+MCU = at90usb1286
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = halfkay
+
+# Build Options
+#   change yes to no to disable
+#
+AUDIO_ENABLE = no           # Audio output on port C6
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+COMMAND_ENABLE = no         # Commands for debug and configuration
+CONSOLE_ENABLE = no         # Console for debug
+EXTRAKEY_ENABLE = no        # Audio control and System control
+MIDI_ENABLE = no            # MIDI support
+MOUSEKEY_ENABLE = no        # Mouse keys
+NKRO_ENABLE = no            # USB Nkey Rollover
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+UNICODEMAP_ENABLE = yes        # Unicode
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

`UNICODEMAP_ENABLE` must not be specified at keyboard level, it requires custom code (the `unicode_map` array) which the Configurator cannot generate; it does not use the `default` keymap.